tests/integration/pyramid_tests.py

```diff
@@ -51,6 +60,22 @@ def setUp(self):
         configurator.add_view(
             local_tracing_within_context, route_name="trace_context", renderer="json")
 
+        mock_filewatcher = mock.Mock(spec=FileWatcher)
+        mock_filewatcher.get_data.return_value = {
+            "secrets": {
+                "jwt/authentication/secret": {
+                    "type": "simple",
+                    "value": self.TOKEN_SECRET,
+                },
+            },
+            "vault": {
+                "token": "test",
+                "url": "http://vault.example.com:8200/",
+            }
+        }
+        secrets = store.SecretsStore("/secrets")
+        secrets._filewatcher = mock_filewatcher
```

Comment:
I just copied this pattern from how the `get_simple` method on the secrets test was setup and run.  I didn't see any pattern for test helpers, but was going to try and consolidate secret creation for tests that need it into something shared rather than this ugly copy pasta that I used here.